### PR TITLE
Adds an admin-spawn Nanotrasen electric guitar

### PR DESCRIPTION
**Pull request by Zonespace27**
[Feature]
## About the PR 
This PR adds a recolor of the bard's electric guitar, made for Nanotrasen, with a fitting blue paint job to boot.
image

Awaits #6800 being merged because I need to mess with some code that PR adds too.

## Why's this needed? 

NT reskins of some nukeop items (esp items as unique as this) is neat, also requested by Ace on discord.

### CODE REVIEW

File: code/obj/item/device/studio_monitor.dm [hunk 1]
```diff
@@ -204,7 +211,10 @@
 			var/mob/living/carbon/human/virtual/V = target
 			. = istype(V.ears, /obj/item/device/radio/headset/syndicate) || istype(V.head, /obj/item/clothing/head/helmet/space/syndicate)
 		else
-			. = istype(target.ears, /obj/item/device/radio/headset/syndicate)
+			if(is_syndicate)
+				. = istype(target.ears, /obj/item/device/radio/headset/syndicate)
+			else
+				. = !istype(target.ears, /obj/item/device/radio/headset/syndicate) //Nanotrasen guitar, Nanotrasen tunes
```
Comment (Zonespace27) [inline code/obj/item/device/studio_monitor.dm:217]:
```suggestion
				. = istype(target.ears, /obj/item/device/radio/headset) //Nanotrasen guitar, Nanotrasen tunes
```
I was dumb when I wrote that

File: code/obj/item/device/studio_monitor.dm [hunk 2]
```diff
@@ -204,7 +211,10 @@
 			var/mob/living/carbon/human/virtual/V = target
 			. = istype(V.ears, /obj/item/device/radio/headset/syndicate) || istype(V.head, /obj/item/clothing/head/helmet/space/syndicate)
 		else
-			. = istype(target.ears, /obj/item/device/radio/headset/syndicate)
+			if(is_syndicate)
+				. = istype(target.ears, /obj/item/device/radio/headset/syndicate)
+			else
+				. = istype(target.ears, /obj/item/device/radio/headset) //Nanotrasen guitar, Nanotrasen tunes
```
Comment (ZeWaka) [inline code/obj/item/device/studio_monitor.dm:217]: don't you mean like a ntso or captain or something headset

Comment (Zonespace27) [reply code/obj/item/device/studio_monitor.dm]: Wasn't sure what it should be set to for a while, settled on a command headset (which also includes NTSOs and such)

**Comment by pali6** [comment]
!merge_upstream

**Pull request merged**